wondrous-app/components/Common/CommentItem/index.tsx

```diff
@@ -64,7 +64,7 @@ const CommentItem = (props) => {
             }}
           />
         )}
-        <div>
+        <Grid flex={1}>
```

Comment:
Is there any difference, except that `<Grid flex={1}>` adds more styles?
it looks the same.

Reply:
There is a difference actually. Long comment text will give the visual difference as in the screenshots below. First one uses a `div` tag as before and the second one uses `<Grid flex={1}>`
1. ![image](https://user-images.githubusercontent.com/35099553/203611720-d70d4f2e-3fe4-40ef-8911-bcbe94541ccb.png)

2. ![image](https://user-images.githubusercontent.com/35099553/203612365-2087b2ae-8b70-477e-8ee0-ecd558c1fae6.png)

Reply:
Great. Thanks for the explanation.